### PR TITLE
[CM-1372] Added Support for static first message 

### PR DESCRIPTION
**Pull request by Sathyan-KM**
## Summary
- Added Support for static first message
- It can be enabled by below code
```
  KMConversationScreenConfiguration.staticTopMessage = "Messages in this chat are encrypted. No one outside this chat can read them"
KMConversationScreenConfiguration.staticTopIcon = "<pass your UIImage>"
```
- By default it is disabled.

## Testing
- Tested by creating new conversation, opening existing conversation and scrolling conversation screen up & down

### CODE REVIEW

File: Sources/Utilities/KMCellConfiguration.swift [hunk 1]
```diff
@@ -11,4 +11,9 @@ public enum KMCellConfiguration {
     
     /// if true then incoming messsage's sender name will be hidden on conversation
     public static var hideSenderName = false
+    
+    public static var staticTopMessage: String = ""
```
Comment (mukesh-komm) [inline Sources/Utilities/KMCellConfiguration.swift:15]: Should it be added to CellConfig when it's at the conv VC level?

File: Sources/ViewModels/ALKConversationViewModel.swift [hunk 2]
```diff
@@ -458,6 +458,8 @@ open class ALKConversationViewModel: NSObject, Localizable {
             }
         case .form:
             return 0
+        case .initialEncryptionMessage:
```
Comment (mukesh-komm) [inline Sources/ViewModels/ALKConversationViewModel.swift:461]: Should it be `initialEncryptionMessage` when it's static top message in other places?

File: Sources/ViewModels/ALKConversationViewModel.swift [hunk 3]
```diff
@@ -1211,8 +1223,13 @@ open class ALKConversationViewModel: NSObject, Localizable {
                 return
             }
             NSLog("messages loaded: ", messages)
-
+
```
Comment (mukesh-komm) [inline Sources/ViewModels/ALKConversationViewModel.swift:1226]: whitespace

File: Sources/Resources/ALKAssets.xcassets/Chat/Chat view/ic_lock.imageset/lock.png
Comment (mukesh-komm) [inline Sources/Resources/ALKAssets.xcassets/Chat/Chat view/ic_lock.imageset/lock.png:1]: Why set this in our SDK when this imageview can be used for other usecases as well? We can leave it empty and ask them to pass it just like the text.

**Review by mukesh-komm — COMMENTED** [review]
Add a changelog entry.

File: Sources/Utilities/KMCellConfiguration.swift (hunk 1 above)
Comment (Sathyan-KM) [reply Sources/Utilities/KMCellConfiguration.swift]: created new file to have this configurations

File: Sources/ViewModels/ALKConversationViewModel.swift (hunk 2 above)
Comment (Sathyan-KM) [reply Sources/ViewModels/ALKConversationViewModel.swift]: changed top statictopmessage

File: Sources/Resources/ALKAssets.xcassets/Chat/Chat view/ic_lock.imageset/lock.png
Comment (Sathyan-KM) [reply Sources/Resources/ALKAssets.xcassets/Chat/Chat view/ic_lock.imageset/lock.png:1]: done

**Review by mukesh-komm — APPROVED** [review]

**Pull request merged**